### PR TITLE
refactor(repo): address walkthrough comments

### PR DESCRIPTION
**Pull request by QZera**
Closes [fa31a13](https://explorer.radicle.gr/nodes/seed.radicle.gr:443/rad:z2BdUVZFvHdxRfdtGJQdSH2kyXNM6/issues/fa31a135845f552972ce9e601c61eead467abc2b)

### CODE REVIEW

File: composables/use-host-app-message.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import * as v from 'valibot'
+import {
+  type IncomingMessage,
+  type OutgoingMessage,
+  incomingMessageSchema,
+} from '~/types/messages'
+
+export function useHostAppMessage() {
```
Comment (maninak) [inline composables/use-host-app-message.ts:8]: Είχαμε πει αυτό

```suggestion
export function useHostAppMessaging() {
```

Source: https://app.radicle.at/nodes/seed.radicle.at/rad:z3Makm6fsQQXmpSFE43DZqwupaEhk/tree/src/utils/webview-messaging.ts

File: stores/tasks.ts [hunk 2]
```diff
@@ -51,10 +52,24 @@ export const useTasksStore = defineStore('tasks', () => {
...
+        const taskIndex = tasks.value.indexOf(task)
+        if (taskIndex === -1) {
+          return
+        }
+
+        const newTask = {
+          ...task,
+          labels,
+          rpb,
+        }
```
Comment (maninak) [inline stores/tasks.ts:69]: We can make these one line and make it much smaller

**Review by maninak — APPROVED** [review]
Niceru! :+1: 

Only found a couple of optional nits. Resolve on your own as needed.

File: stores/tasks.ts (hunk 2 above)
Comment (QZera) [reply stores/tasks.ts]: Done. Found a couple more case of this and reduced them also.

**Pull request merged**